app/models/scheme.rb

```diff
@@ -228,4 +230,8 @@ def validate_confirmed
       end
     end
   end
+
+  def strip_whitespace
+    self.service_name = service_name.strip unless service_name.nil?
```

Comment:
I wonder if this is worth genericising to all fields that respond to strip/squish and extracting to `ApplicationRecord`?

Reply:
Yeah, possibly, will have a look